packages/p2p/stores/order-details-store.js

```diff
@@ -0,0 +1,97 @@
+import { observable, action } from 'mobx';
+import { requestWS } from 'Utils/websocket';
+import { secondsToTimer } from 'Utils/date-time';
+import ServerTime from 'Utils/server-time';
+import { localize } from 'Components/i18next';
+
+export default class OrderDetailsStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        this.general_store = this.root_store.general_store;
+        this.order_store = this.root_store.order_store;
```

Comment:
In general, try to access other stores by going through `root_store` rather than defining other stores on the `this` object directly. Doing this will only add extra maintenance, i.e. when we're adding new stores we need to remember to set these every store.